examples/js/controls/TrackballControls.js

```diff
@@ -29,7 +29,7 @@ THREE.TrackballControls = function ( object, domElement ) {
 	this.minDistance = 0;
 	this.maxDistance = Infinity;
 
-	this.keys = [ 65 /*A*/, 83 /*S*/, 68 /*D*/ ];
+	this.keys = [ 'KeyA' /*A*/, 'KeyS' /*S*/, 'KeyD' /*D*/ ];
```

Comment:
How does this change affect french, german, ... keyboards?

Reply:
see https://github.com/mrdoob/three.js/pull/21056#issue-552749356